Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -777,24 +771,28 @@ public async Task<List<ChannelInfo>> GetChannels(ListingsProviderInfo info, Canc
             using var options = new HttpRequestMessage(HttpMethod.Get, ApiUrl + "/lineups/" + listingsId);
             options.Headers.TryAddWithoutValidation("token", token);
 
-            var list = new List<ChannelInfo>();
-
             using var httpResponse = await Send(options, true, info, cancellationToken).ConfigureAwait(false);
             await using var stream = await httpResponse.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false);
             var root = await _jsonSerializer.DeserializeFromStreamAsync<ScheduleDirect.Channel>(stream).ConfigureAwait(false);
             _logger.LogInformation("Found {ChannelCount} channels on the lineup on ScheduleDirect", root.map.Count);
             _logger.LogInformation("Mapping Stations to Channel");
 
-            var allStations = root.stations ?? Enumerable.Empty<ScheduleDirect.Station>();
+            var allStations = root.stations ?? new List<ScheduleDirect.Station>();
```

Comment:
Why?

Reply:
Otherwise `allStations` is an `IEnumerable` instead of a `List`, so we can't use `List` specific functions